pages/common/mlr.md

```diff
@@ -0,0 +1,23 @@
+# mlr
+
+> Miller is like awk, sed, cut, join, and sort for name-indexed data such as CSV, TSV, and tabular JSON.
+
+- Pretty-print a CSV file in a tabular format:
+
+`mlr --icsv --opprint cat {{example.csv}}`
+
+- Sort first alphabetically on field1, and then sort in descending numerical order on field2:
+
+`mlr --icsv --opprint sort -f {{field1}} -nr {{field2}} {{example.csv}}`
```

Comment:
I think we can split this into 2 examples. Rather than putting both options in a single one. We introduce each options slowly and then build up to more complex examples.